integration/single-node-slug-deploy/check.go

```diff
@@ -391,7 +396,39 @@ func verifyHealthChecks(config *preparer.PreparerConfig, services []string) erro
 		}
 	}
 
+	res, err := store.GetServiceHealth(services[1])
+	getres, _ := store.GetHealth(services[1], name)
+	if err != nil {
+		return err
+	}
+	val := res[fmt.Sprintf("health/p2-preparer/%s", name)]
+	if getres.Id != val.Id || getres.Service != val.Service || getres.Status != val.Status {
+		return fmt.Errorf("GetServiceHealth failed %+v", res)
+	}
+
 	// if it reaches here it means health checks
 	// are being written to the KV store properly
 	return nil
 }
+
+func emptyHealthCheck(config *preparer.PreparerConfig) error {
```

Comment:
Consul has a package called testutil that would be very good for tests that require consul API calls. Using the consul package is probably the easiest way to write these tests.